reference/Upload-API.yaml

```diff
@@ -879,5 +879,299 @@ paths:
                       etag: 55a64df551425fcc55e4d42a148795d9f25f89d6
         description: The request body should contain a reference to each part along with the Etag of each upload.
     parameters: []
+  '/api/v1/{apiUserId}/{apiKey}/{targetId}/{languageCode}/search':
+    parameters:
+      - schema:
+          type: string
+        name: apiUserId
+        in: path
+        required: true
+        description: Guid identifying the requesting client
+      - schema:
+          type: string
+        name: apiKey
+        in: path
+        required: true
+        description: Second layer encrypted identifier for the requesting client
+      - schema:
+          type: string
+        name: targetId
+        in: path
+        description: Guid identifying the target where articles are allowed to be published
+      - schema:
+          type: string
+        name: languageCode
+        in: path
+        required: true
+        description: Two characters ISO language code
+    get:
+      summary: Article search
+      tags: []
```

Comment:
It'd be nice to add tag named 'seach' here and top of the spec document, so that we'll see a section named `search` on the left hand side of doc
![Screenshot 2022-07-01 at 09 10 16](https://user-images.githubusercontent.com/8457942/176878311-6c013361-bddd-43b7-9ca5-5f7e2efb9176.png)

Reply:
Added